hdfs3/core.py

```diff
@@ -24,7 +24,7 @@ def hdfs_conf(conf_dir=None):
     """ Load HDFS config from default locations. """
     if conf_dir == None:
         confd = os.environ.get('HADOOP_CONF_DIR', os.environ.get('HADOOP_INSTALL',
-                           '') + '/hadoop/conf')
+                           '/etc') + '/hadoop/conf')
```

Comment:
No I'm not sure, since we normally expect the HADOOP env vars to be set. It is conceivable that the default has changed through time. I am OK to include your change, but I would also like a more fool-proof way, if possible.
Of course, on option would have been to error if the variables are not set or a file not found, but in the code now we error only later, when trying the default hdfs port on localhost fails.